electron/src/window/AboutWindow.ts

```diff
@@ -18,13 +18,13 @@
  */
 
 import {BrowserWindow, IpcMessageEvent, app, ipcMain, session, shell} from 'electron';
-import fileUrl = require('file-url');
 import * as path from 'path';
 
-import {i18nLanguageIdentifier} from '../interfaces/';
+import fileUrl = require('file-url');
```

Comment:
This should work, too:

```suggestion
import fileUrl from 'file-url';
```